### PR TITLE
Arghh06 kuching, sarawak

### PR DESCRIPTION
**Pull request by Arghh06**
---
githubHandle: Arghh06
latitude: 1.544765
longitude: 110.365219
---

### CODE REVIEW

File: _pins/Arghh06.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: Arghh06
+latitude: 1.6077° N
+longitude: 110.3785° E
```
Comment (leiguoyang) [inline _pins/Arghh06.json]: @Arghh06 You are so close to have your pin on the map. 😅 As the Travis CI says you are missing document start "---".  I suggest you add `---` at the start and at the end of this file. Second, remove the `° N` and the `° E` in line 2 and line 3 respectively.  

error-screenshot_0003

File: _pins/arghh06.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+githubhandle: arghh06
+lattitude: 1.544765
+longitude: 110.365219
```
Comment (leiguoyang) [inline _pins/arghh06.json]: and I suggest you delete this file because you've have one above.

Comment (Arghh06) [inline _pins/arghh06.json]: @leiguoyang , I have been trying to delete this file all the while but until now I can't find the way to delete it.

**Comment by Arghh06** [comment]
---
githubHandle: Arghh06
latitude: 1.544765
longitude: 110.365219
---

**Pull request merged**